src/form.ts

```diff
@@ -41,10 +41,10 @@ export class Form {
     i: number
     e: HTMLElement
     resolve: (value) => void
-    reject: (reason?) => void
     fields: Fields
     results: Results
     static activeForm = false
+	static disposeCurrent: () => void
```

Comment:
This is a "constant" function , better defined it the typescript way. Also, we need a better name, so please change to:

```typescript
// closes the active form 
static close() {
   ...
}
```